pytorch_seq2seq/service.py

```diff
@@ -0,0 +1,45 @@
+import typing as t
+import unicodedata
+import re
+
+import bentoml
+from bentoml.io import Text
+
+
+def unicodeToAscii(s):
+    return ''.join(
+        c for c in unicodedata.normalize('NFD', s)
+        if unicodedata.category(c) != 'Mn'
+    )
+
+
+def normalizeString(s):
+    s = unicodeToAscii(s.lower().strip())
+    s = re.sub(r"([.!?])", r" \1", s)
+    s = re.sub(r"[^a-zA-Z.!?]+", r" ", s)
+    return s
+
+
+encoder = bentoml.pytorch.load_runner(
+    "pytorch_tldr_encoder"
+)
+
+decoder = bentoml.pytorch.load_runner(
+    "pytorch_tldr_decoder"
+)
+
+svc = bentoml.Service(
+    name="pytorch_tldr_demo",
+    runners=[
+        encoder,
+        decoder,
+    ],
+)
+
+
+@svc.api(input=Text(), output=Text())
+def predict(input_arr: t.List[str]) -> t.List[str]:
+    input_arr = list(map(normalizeString, input_arr))
+    enc_arr = encoder.run_batch(input_arr)
+    res = decoder.run_batch(enc_arr)
+    return res[0]['generated_text']
```

Comment:
```suggestion
@svc.api(input=Text(), output=Text())
def predict(input_arr: str) -> str:
    input_arr = normalizeString(input_arr)
    enc_arr = encoder.run(input_arr)
    res = decoder.run(enc_arr)
    return res['generated_text']
```

Thanks for the great work.

The use case in the gallery would help us build a better and easier-to-use API of bentoml.

Here are some suggestions from me:

* use `run` rather than `run_batch` if possible (users will get simpler code and the ability of dynamic batching)
* use `async def predict` rather than `def predict` to get better performance.